### PR TITLE
Add numerical battery percentage and voltage to Debug Menu

### PR DESCRIPTION
**Pull request by Cralex**

### CODE REVIEW

File: components/retro-go/rg_gui.c [hunk 1]
```diff
@@ -1266,6 +1269,15 @@ void rg_gui_debug_menu(const rg_gui_option_t *extra_options)
...
     snprintf(block_free, 20, "%d+%d", stats.freeBlockInt, stats.freeBlockExt);
     snprintf(uptime, 20, "%ds", (int)(rg_system_timer() / 1000000));
+    float batteryPct, batteryVlt;
+    if (rg_input_read_battery(&batteryPct, &batteryVlt))
+    {
+        snprintf(battery_info, sizeof(battery_info), "%.2f%% | %.2fV", batteryPct, batteryVlt);
+    }
+    else
+    {
+        snprintf(battery_info, sizeof(battery_info), "Battery: N/A");
```
Comment (ducalex) [inline components/retro-go/rg_gui.c:1279]: This will end up displaying Battery: Battery: N/A, no? I think the string should just be "N/A".

**Comment by Cralex** [comment]
Sorry, I went deep in the rabbit hole trying things out and then life happened. I agree on that change. :)

**Comment by ducalex** [comment]
I don't have any ability to test right now, can you confirm this is good to merge? 

Thanks!

**Comment by Cralex** [comment]
It looks good on my end. I'm running firmware with this PR on my test devices.

**Pull request merged**